client/lib/explat/internals/naive-client-side-rollout.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Internal dependencies
+ */
+import user from 'calypso/lib/user';
+
+/**
+ * Naively ship a feature to a percentage of our users.
+ * - Very basic, this exists to replace the last remaining calypso-abtest use-case.
+ * - Works best for logged-in users.
+ * - Uses randomisation for logged out users.
+ * - Won't work across the log-in boundary.
+ *
+ * @param percentage 0-100
+ * @returns Boolean of whether the user should have the feature enabled, i.e. true = rolled out, false = not rolled out.
+ */
+export default function naiveClientSideRollout( percentage: number ): boolean {
+	const maybeUserId = user()?.get()?.ID;
+
+	// zero-indexed buckets: 0-99
+	let bucket;
+	if ( maybeUserId ) {
+		bucket = Number( maybeUserId ) % 100;
+	} else {
+		bucket = Math.random() * 100;
```

Comment:
Should we attempt to use the `tk_ai` cookie as the user ID if it exists?

Reply:
Hashing is a problem here, but what we can do is store the assignment in localStorage

Reply:
Yeah, I don't think it's worth worth complicating things with storage calls.